### PR TITLE
feat: HeadArticleSearch

### PR DESCRIPTION
**Pull request by Vladosik12333**
dev
## GitHub Project

* [Jira Project ticket](https://localhost-vlad.atlassian.net/jira/software/projects/EEF/boards/3?assignee=60df293aa3de4a006ba94839&selectedIssue=EEF-59)

## Code reviewers

- [ ] All

### Second Level Review

- [ ] @VadymKharyton | @andrii-kuzmichov 

## Summary of issue

Task about search on slider

## Summary of change

changes some files

## Testing approach

Yes, in browser.

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/constants/HeadArticleSearchConstants.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const headArticleSearchConstants = {
```
Comment (andrii-kuzmichov) [inline src/constants/HeadArticleSearchConstants.js]: use constants format LIKE_THAT

File: src/components/landing/HeadArticleButtons/HeadArticleButtons.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import AuthComponent from "../../../security/authComponent";
+import ModalWind from "../../modal-wind";
+import constants from "../../../constants/HeadArticleConstants";
```
Comment (andrii-kuzmichov) [inline src/components/landing/HeadArticleButtons/HeadArticleButtons.jsx:5]: do not use for import

File: src/constants/HeadArticleSearchConstants.js (hunk 1 above)
Comment (Vladosik12333) [reply src/constants/HeadArticleSearchConstants.js]: /+/

File: src/components/landing/HeadArticleButtons/HeadArticleButtons.jsx (hunk 2 above)
Comment (Vladosik12333) [reply src/components/landing/HeadArticleButtons/HeadArticleButtons.jsx:5]: /+/

**Review by akuzmichov-amplify — APPROVED** [review]

**Review by andrii-kuzmichov — APPROVED** [review]

**Pull request merged**